Sources/Kitura/RouterResponse.swift

```diff
@@ -179,6 +179,143 @@ public class RouterResponse {
+            return CookieAttribute(_CookieAttribute.portList(value))
+        }
+
+        /// The cookie’s expiration date. The expiration date is the date when the cookie should be deleted.
+        public static func expires(_ value: Date?) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.expires(value))
+        }
+
+        /// A value stating how long in seconds the cookie should be kept, at most.
+        public static func maximumAge(_ value: String) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.maximumAge(value))
+        }
+
+        /// The URL that set this cookie.
+        public static func originURL(_ value: URL?) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.originURL(value))
+        }
+
+        /// The version of the cookie. Must be either 0 or 1. The default is 0.
+        public static func version(_ value: Int) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.version(value))
+        }
+
+        /// A String value representing a boolean (TRUE/FALSE), stating whether the cookie should be discarded at the end of the session.
+        public static func discard(_ value: String) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.discard(value))
+        }
+
+        /// A  boolean value that indicates whether this cookie should only be sent over secure channels.
+        public static func isSecure(_ value: Bool) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.isSecure(value))
+        }
+
+        /// A comment for the cookie.
+        public static func comment(_ value: String?) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.comment(value))
+        }
+
+        /// A URL that can be presented to the user as a link for further information about this cookie.
+        public static func commentURL(_ value: String?) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.commentURL(value))
+        }
+
+        /// Custom cookie attributes
+        ///
+        ///Note: Custom cookie attributes are not honoured by Foundation yet.
+        public static func custom(_ key: String, _ value: String) -> CookieAttribute {
+            return CookieAttribute(_CookieAttribute.custom(key, value))
+        }
+    }
+
+    /// Add a cookie to the response.
+    ///
+    /// This function creates an `HTTPCookie`  from the provided attributes and adds it to the `cookies` dictionary.
+    /// - Parameter name: The cookie’s name.
+    /// - Parameter value: The cookie‘s value.
+    /// - Parameter domain: The domain of the cookie.
+    /// - Parameter path: The cookie’s path.
+    /// - Parameter otherAttributes: An array of  any other optional cookie attributes
+    public func addCookie(name: String, value: String, domain: String, path: String, otherAttributes: [CookieAttribute] = []) {
```

Comment:
This will allocate an array for `otherAttributes` every time, even if it's not used and is empty. It would be better to make it an optional parameter with a default of `nil`.